spec/features/admin/adjustments_spec.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
-require "system_helper"
+require "spec_helper"
 
-describe '
+feature '
```

Comment:
This change doesn't need to be reverted. You can actually rename all `feature` to `describe`. It's the same. Someone just had the idea that `feature` sounds nicer and introduced it as an alias for `describe`.

Reply:
I'm not so sure about this, lets take `spec/features/admin/multilingual_spec.rb` as an example:

- with 'spec_helper' - using `feature` (L5) - using `background` (L11) - spec passes 
- with 'spec_helper' - using `describe` (L5) - using `background` (L11) - spec fails
- with 'spec_helper' - using `describe` (L5) - using `before` (L11) - spec passes

It seems to make a difference.